Packs/CheckpointFirewall/Integrations/CheckPointFirewallV2/CheckPointFirewallV2.py

```diff
@@ -67,7 +75,7 @@ def test_connection(self):
         Otherwise, an exception should be raised by self._http_request()
         """
         response = self._http_request(method='POST', url_suffix='show-api-versions', headers=self.headers,
-                                      ok_codes=(200, 500), resp_type='response')  # returns requests.Response
+                                      ok_codes=(200, 500), resp_type='response', json_data={})
```

Comment:
why did you add `json_data` did you intend to pass it to `self._http_request`?

Reply:
Similar to the issue we had with `login()`, using a json that isn't explicitly empty seems to be considered invalid on the FW side.